src/utils/Configuration.ts

```diff
@@ -2,15 +2,19 @@
 import * as yml from "node-yaml";
 import { IInvokeConfig, IMOTConfig, IS3Config } from "../models";
 import {ERRORS} from "../models/Enums";
+import SecretsManager, {GetSecretValueRequest, GetSecretValueResponse} from "aws-sdk/clients/secretsmanager";
+import {ISecret} from "../models/ISecret";
 
 /**
  * Configuration class for retrieving project config
  */
 class Configuration {
   private static instance: Configuration;
   private readonly config: any;
+  private readonly secretsClient: SecretsManager;
 
   private constructor(configPath: string) {
+    this.secretsClient = new SecretsManager({ region: "eu-west-1" });
```

Comment:
Has the region alway been hard coded? I'm not sure if want this to be hard coded for privacy and security reasons maybe?

Reply:
All I did was move this from the Service into Configuration, it was there before and is consistent with the secret retrieval logic in other services.